label_studio/tasks/models.py

```diff
@@ -798,6 +798,7 @@ class TaskLock(models.Model):
         on_delete=models.CASCADE,
         help_text='User who locked this task',
     )
+    created_at = models.DateTimeField(_('created at'), auto_now_add=True, help_text='Creation time', null=True)
```

Comment:
I believe auto_now_add will set created_at to the current time for all existing rows, and it'll do so without using a db default, making this an unsafe migration (and also the null=True will have no effect)

Reply:
Fixed.